lib/SIL/Utils/MemAccessUtils.cpp

```diff
@@ -400,12 +400,7 @@ bool swift::isLetAddress(SILValue address) {
 //                      MARK: Deinitialization barriers.
 //===----------------------------------------------------------------------===//
 
-static bool isBarrierApply(FullApplySite) {
```

Comment:
For the reasons discussed above, you can remove this utilities in C++.

Reply:
As discussed elsewhere, mayAccessPointer remains in C++.